source/sass/blocks/maps-page.scss

```
@@ -0,0 +1,13 @@
+.maps-page {
+  width: 100%;
+}
+
+.maps-page__widtch {
+  @media (min-width: 600px) {
```

Всё таки может быть вот так? 😏
```suggestion
  @media (min-width: 660px) {
```